### PR TITLE
Implement Hyperledger Fabric specific CouchDB chart

### PR DESCRIPTION
**Pull request by alexvicegrab**
Signed-off-by: Alejandro Vicente Grabovetsky <sasha@aid.technology>

**What this PR does / why we need it**:

Implements Hyperledger Fabric node hosting custom CouchDB instance for holding Blockchain data.

**Special notes for your reviewer**:

A future chart (hlf-peer) will sometimes make use of this chart.

### CODE REVIEW

File: incubator/hlf-couchdb/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1beta2
```
Comment (unguiculus) [inline incubator/hlf-couchdb/templates/deployment.yaml]: `apps/v1`

**Review by unguiculus — CHANGES_REQUESTED** [review]
Please follow our best practices for persistence: 
https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md#persistence

**Comment by unguiculus** [comment]
I'd suggest you move the chart to stable directly.

**Comment by alexvicegrab** [comment]
Hi @unguiculus, thank you for reviewing the chart and suggesting improvements. Please review at your convenience!

**Review by unguiculus — APPROVED** [review]

**Pull request merged**